src/legendhpges/base.py

```diff
@@ -105,6 +110,65 @@ def _decode_polycone_coord(self) -> tuple[list[float], list[float]]:
         Must be overloaded by derived classes.
         """
 
+    def distance_to_surface(
+        self, coords: np.ndarray | list, surface_indices: list | None = None
+    ) -> np.ndarray:
```

Comment:
```suggestion
    ) -> NDArray:
```
https://numpy.org/devdocs/reference/typing.html#numpy.typing.NDArray